### PR TITLE
ID translation for pod topology spread constraints

### PR DESCRIPTION
**Pull request by wahyuoi**
address #20111 

translation for page /docs/concepts/workloads/pods/pod-topology-spread-constraints/

### CODE REVIEW

**Comment by wahyuoi** [comment]
hi @girikuncoro, PTAL
I just push an update

File: content/id/docs/concepts/workloads/pods/pod-topology-spread-constraints.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+title: Batasan Persebaran Topologi Pod
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.18" state="beta" >}}
+
+Kamu dapat menggunakan batasan perseberan topologi (_topology spread constraints_)
```
Comment (girikuncoro) [inline content/id/docs/concepts/workloads/pods/pod-topology-spread-constraints.md:11]: typo:
```suggestion
Kamu dapat menggunakan batasan persebaran topologi (_topology spread constraints_)
```

File: content/id/docs/concepts/workloads/pods/pod-topology-spread-constraints.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+Di Kubernetes, arahan yang terkait dengan "Afinitas" mengontrol bagaimana Pod dijadwalkan - 
+lebih terkumpul atau lebih tersebar.
+
+- Untuk `PodAffinity`, kamu dapat mencoba mengumpulkan beberapa Pod ke dalam suatu
+domain topologi yang memenuhi syarat.
+- Untuk `PodAntiAffinity`, hanya satu Pod yang dalam dijadwalkan pada sebuah domain topologi.
+
+Fitur "EvenPodsSpread" memberikan opsi fleksibilas untuk mendistribusikan Pod secara merata
+pada domain topologi yang berbeda, untuk meraih ketersediaan yang tinggi atau menghemat biaya.
+Ini juga dapat membantu saat perbaruan bergilir dan menaikan jumlah replika dengan lancar.
```
Comment (girikuncoro) [inline content/id/docs/concepts/workloads/pods/pod-topology-spread-constraints.md:280]: typo
```suggestion
Ini juga dapat membantu saat perbaruan bergilir dan menaikkan jumlah replika dengan lancar.
```

File: content/id/docs/concepts/workloads/pods/pod-topology-spread-constraints.md [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+Silakan baca [motivasi](https://github.com/kubernetes/enhancements/blob/master/keps/sig-scheduling/20190221-even-pods-spreading.md#motivation) untuk lebih detail.
```
Comment (girikuncoro) [inline content/id/docs/concepts/workloads/pods/pod-topology-spread-constraints.md:281]: typo
```suggestion
Silahkan baca [motivasi](https://github.com/kubernetes/enhancements/blob/master/keps/sig-scheduling/20190221-even-pods-spreading.md#motivation) untuk lebih detail.
```

**Pull request merged**